lib/index.ts

```diff
@@ -131,6 +130,44 @@ class LogService {
     }
   }
 
+  addLogger(
+    loggerScopeFunction: (config: InitialConfig) => LoggerPlugin,
+    pluginConfig: PluginConfig) {
+    if (typeof loggerScopeFunction !== 'function')
+      return;
```

Comment:
Not need to check if the parameter is a function, if its not it should throw an exception when trying to call `loggerScopeFunction` . If it only return, the developer will think that it's all ok, and will be harder to debug 'cause none exception will be throw in any point of code.

Reply:
Good, I'm going to remove it